app/assets/stylesheets/application.sass

```diff
@@ -48,3 +57,87 @@ p
 .wf-active h3,
 .wf-active p
   visibility: visible
+
+.hover-options
+  position: absolute
+  top: 2px
+  right: -1px
+  display: none
+  text-align: right
+  z-index: 1000
+
+.has-hover
+  &:hover, &:focus
+    > .hover-options
+      display: block
+
+body > footer
+  @media(min-width: 768px)
+    +align-items(stretch)
+    +display(flex)
+    +flex-direction(row)
+    +justify-content(flex-start)
+    &.bg-dark
+      padding: 0
+
+  h1.brand
+    font-size: 7rem
+    line-height: 4.5rem
+    letter-spacing: -1rem
+    margin: 3rem 0
+    @media(min-width: 768px)
+      +flex(1)
+      font-size: 6vw
```

Comment:
Wait – they were already here? Then I should perhaps peruse my pull requests as closely as this one more often. :wink: